lib/Translator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+class Translator
+{
+    private const DEFAULT_LANGUAGE = 'english';
+
+    public static function loadTranslations(): void
+    {
+        global $CONFIG, $_LANG;
+
+        if (isset($_SESSION['Language'])) {
+            $language = $_SESSION['Language'];
+        } elseif (isset($_SESSION['adminlang'])) {
+            $language = $_SESSION['adminlang'];
+        } else {
+            $language = $CONFIG['Language'];
+        }
```

Comment:
Depdends on the PHP version you could use
```suggestion
        $language = $_SESSION['Language'] ?? $_SESSION['adminlang'] ?? $CONFIG['Language'];
```

Reply:
Fixed